mysql/8.0.32/TAG

```diff
@@ -0,0 +1 @@
+8.0.32.1
```

Comment:
should be 8.0.32.<b>2</b> in this case, isn't it?

Reply:
`mysql:8.0.32.1` is not broken and still exist on quay.io. So it need not rebuild.